esp-link/cgiwifi.c

```diff
@@ -121,10 +121,11 @@ static ICACHE_FLASH_ATTR
 void wifiStartMDNS(struct ip_addr ip) {
   if (!mdns_started) {
     os_memset(&mdns_info, 0, sizeof(struct mdns_info));
-    mdns_info.host_name = flashConfig.hostname;
+    mdns_info.host_name = flashConfig.hostname,
```

Comment:
Let's try the zalloc version to see whether that fixes the issues I'm seeing with mdns..